Resources/Textures/Structures/Power/substation.rsi/meta.json

```diff
@@ -1,86 +1,108 @@
 {
-  "version": 1,
-  "license": "CC0-1.0",
-  "copyright": "Created by EmoGarbage404 (github)",
-  "size": {
-    "x": 32,
-    "y": 32
-  },
-  "states": [
-    {
-      "name": "substation"
+    "version": 1,
+    "license": "CC0-1.0",
+    "copyright": "Created by EmoGarbage404 (github), screen sprites created by joshepvodka (github) and updated by AwareFoxy(github)",
```

Comment:
Have you asked joshepvodka if they were the one that created the screen sprites, and that they are okay with relicensing to CC0? (In their PR they are licensed as CC-BY-SA)